### ISSUE TITLE
Add option to stop 'execute' once it submits TF requests 

### ISSUE THREAD

**Issue by lukaszachy** [RedHatQE/newa#142]
Use case: Start the testing via TF but leave it running unattended, eg.
```
$ newa event -e ID .... execute --no-wait
$ newa --prev-state-dir execute --continue report
```

**Comment by kkaarreell** [comment]
Currently, we have a default limit for number of workers. NEWA won't therefore schedule all TF requests at once.
What should be the behavior when limit for workers is smaller than the number of requests?

1. drop the limit from the code entirely
2. ignore the limit when --no-wait
3. inform a user that the limit is in effect
4. do nothing extra and leave user surprised that --continue is eventually scheduling new requests

**Comment by lukaszachy** [comment]
I'd say that `drop the limit from the code entirely` effectively. 

But I would already expect newa to behave that way, because 'no-wait' means worker has finished the job once it submitted the TF request and should be able to pick next job. Thus exhausting all 'to be scheduled' jobs

**Comment by kkaarreell** [comment]
> I'd say that `drop the limit from the code entirely` effectively.
> 
> But I would already expect newa to behave that way, because 'no-wait' means worker has finished the job once it submitted the TF request and should be able to pick next job. Thus exhausting all 'to be scheduled' jobs

good point

**Comment by kkaarreell** [comment]
> > I'd say that `drop the limit from the code entirely` effectively.

Let's discuss it on the next wg mtg.

> > But I would already expect newa to behave that way, because 'no-wait' means worker has finished the job once it submitted the TF request and should be able to pick next job. Thus exhausting all 'to be scheduled' jobs

 good point

**Comment by kkaarreell** [comment]
WIP https://github.com/RedHatQE/newa/pull/144

**Comment by kkaarreell** [comment]
Implemented through #144
